pkgs/tools/security/swtpm/default.nix

```diff
@@ -61,7 +66,7 @@ stdenv.mkDerivation rec {
         '# define CERTTOOL_NAME "${gnutls}/bin/certtool"'
   '';
 
-  doCheck = true;
+  doCheck = stdenv.isLinux;
```

Comment:
is thats because cuse tests fails on mac?
Could we disable those instead?

Reply:
I'll try to fix the tests.

Reply:
For whatever it's worth, I'm able to use this build of swtpm to successfully still boot/run Windows 11 image via QEMU, so that use case does work.